classes/Sensors/System.php

```diff
@@ -299,7 +299,7 @@ public function WPUpdate($automatic)
      */
     public function LogFilesPruning()
     {
-        if ($this->plugin->GetGlobalOption('purge-log', 0)) {
+        if ($this->plugin->GetGlobalOption('purge-404-log', 0)) {
```

Comment:
@joseforland in the name of the option there is already the prefix so the name is: **wsal-purge-404-log**